src/gnosisSafe/getWeb3.js

```diff
@@ -0,0 +1,109 @@
+import Web3 from "web3";
```

Comment:
Can we refactor this file a bit more?

Reply:
Not sure exactly, could you elaborate a bit more?